frontend/src/scenes/APM/apmLogic.ts

```diff
@@ -0,0 +1,123 @@
+import { kea } from 'kea'
+import { EventType, FilterType, PropertyOperator } from '~/types'
+import api from 'lib/api'
+
+import { apmLogicType } from './apmLogicType'
+import { getChartColors } from 'lib/colors'
+
+const eventApiProps: Partial<FilterType> = {
+    properties: [
+        { key: '$performance_pageLoaded', value: '0', operator: PropertyOperator.GreaterThan, type: 'event' },
+        { key: '$performance_raw', value: 'is_set', operator: PropertyOperator.IsSet, type: 'event' },
```

Comment:
I'd be tempted to leave that for the time being. To avoid a release of the SDK...

I was seeing it as majorPart_minorPart

Am sure well need to revisit if we do carry on with this and could change then?

Reply:
Oh, which we'll have to do for the 64kb limit thing 🙈